Base VIPER Interfaces/BaseWireframe.swift

```diff
@@ -3,14 +3,14 @@ import UIKit
 protocol WireframeInterface: class {
 }
 
-class BaseWireframe {
+class BaseWireframe<View> where View: UIViewController {
```

Comment:
It's a really small nitpick, but I would like to keep the "controller" in the naming, to avoid this being a bit misleading. If we don't want to take the name `ViewController` since it tends to be used, can we use something like `WireframeViewController` or something similar to that, perhaps?

Maybe the others end up liking the naming though, so treat this as my 2 cents about the change. :)

Reply:
We can go with `ViewController` 👍 
It's generic here, so it shouldn't colide with any other created ViewController